plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -448,21 +451,20 @@ private void updatePartitionStatisticsBatch(Table table, Map<String, Function<Pa
     public List<String> getAllTables(String databaseName)
     {
         try {
-            return stats.getGetAllTables().call(() -> {
-                List<String> tableNames = getPaginatedResults(
-                        glueClient::getTables,
-                        new GetTablesRequest()
-                                .withCatalogId(catalogId)
-                                .withDatabaseName(databaseName),
-                        GetTablesRequest::setNextToken,
-                        GetTablesResult::getNextToken)
-                        .map(GetTablesResult::getTableList)
-                        .flatMap(List::stream)
-                        .filter(tableFilter)
-                        .map(com.amazonaws.services.glue.model.Table::getName)
-                        .collect(toImmutableList());
-                return tableNames;
-            });
+            List<String> tableNames = getPaginatedResults(
+                    glueClient::getTables,
+                    new GetTablesRequest()
+                            .withCatalogId(catalogId)
+                            .withDatabaseName(databaseName),
+                    GetTablesRequest::setNextToken,
+                    GetTablesResult::getNextToken,
+                    stats.getGetAllTables())
```

Comment:
I think we should merge stats couneters for getting views and tables.

Reply:
Can you also rename `dropDatabase` stats counter t`deleteDatabase`.

(you can put all the renames in single commit)

Reply:
Same for `renameDatabase` -> `updateDatabase`

Reply:
And `dropTable` -> `deleteTable`

Reply:
`batchGetPartitionAsync` is not covered with stats from what I see.

Reply:
`dropPartition` -> `deletePartition`